api/sds.proto

```diff
@@ -31,8 +31,8 @@ message DataSource {
     // Local filesystem data source.
     string filename = 1 [(validate.rules).string.min_bytes = 1];
 
-    // [#not-implemented-hide:]
-    bytes inline = 2;
+    // Bytes inlined in the configuration.
+    bytes inline = 2 [(validate.rules).string.min_bytes = 1];
```

Comment:
You want the validation rule for bytes, rather than strings here (see https://github.com/lyft/protoc-gen-validate/blob/master/validate/validate.proto#L50).

Reply:
Fixed... But the validator doesn't workaround `inline` keyword, so it's broken anyway :(